ingest/src/main/java/com/microsoft/azure/kusto/ingest/source/BlobSourceInfo.java

```diff
@@ -8,6 +8,8 @@
 public class BlobSourceInfo extends AbstractSourceInfo {
 
     private String blobPath;
+    private String blobPathWithoutSecrets;
```

Comment:
I suggest not "saving" this property in the object since it's just a simple calculation that doesn't cost much. So maybe just do the calculation in a method on demand...